src/main/java/org/kuali/kfs/kim/service/impl/UiDocumentServiceImpl.java

```diff
@@ -771,8 +771,8 @@ protected List<RoleMember> populateRoleMembers(
                         if (roleMember.getActiveFromDate() != null) {
                             roleMemberImpl.setActiveFromDateValue(
                                     new java.sql.Timestamp(roleMember.getActiveFromDate().getTime()));
-                        } else {
-                            // CU Customization KFSPTS-27162 Set default for activeFromDate field
+                        } else if(!roleMember.isEdit()){
+                            // CU Customization If this is a new role and activeFromDate is null, set default for activeFromDate field
                             roleMemberImpl.setActiveFromDateValue(dateTimeService.getCurrentTimestamp());
```

Comment:
This code appears to be affecting new role members that are not necessarily on a new role. Please update this comment for clarity.

Reply:
updated